airflow/models/dag.py

```diff
@@ -52,6 +53,8 @@
 from airflow.utils.sqlalchemy import UtcDateTime, Interval
 from airflow.utils.state import State
 
+DAGCACHED_ENABLED = configuration.getboolean('core', 'dagcached', fallback=True)
```

Comment:
We should probably just have this setting at one place (May be in `settings.py`) and import it from there.

Reply:
WDYT?

Reply:
Good point. I will change this

Reply:
Done.